cylc/flow/network/multi.py

```diff
@@ -107,4 +107,4 @@ def _report_single(report, workflow, result):
 
 
 def _report(_):
-    print('Command submitted; the scheduler will log any problems.')
+    pass
```

Comment:
Is this client-side "reporting" needed for anything?

Reply:
Not really, the multi-command reporting is more important due to the potential for some, but not all commands to fail.

Reply:
I'll leave it there for now, in case we think of a use for it.

Reply:
The plan was to do something fancy - https://github.com/cylc/cylc-flow/issues/4778

One day...